src/app/pages/RoflAppInstanceDetailsPage/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { FC } from 'react'
+import { useHref, useParams } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import Typography from '@mui/material/Typography'
+import { Layer, RoflInstance, useGetRuntimeRoflAppsIdInstances } from '../../../oasis-nexus/api'
+import { SearchScope } from '../../../types/searchScope'
+import { AppErrors } from '../../../types/errors'
+import { API_MAX_TOTAL_COUNT } from '../../config'
+import { RoflAppInstanceDetailsContext } from './hooks'
+import { useRequiredScopeParam } from '../../hooks/useScopeParam'
+import { useScreenSize } from '../../hooks/useScreensize'
+import { PageLayout } from '../../components/PageLayout'
+import { SubPageCard } from '../../components/SubPageCard'
+import { TextSkeleton } from '../../components/Skeleton'
+import { StyledDescriptionList } from '../../components/StyledDescriptionList'
+import { RouterTabs } from '../../components/RouterTabs'
+import { RoflAppLink } from '../../components/Rofl/RoflAppLink'
+import { CopyToClipboard } from '../../components/CopyToClipboard'
+import { useTypedSearchParam } from '../../hooks/useTypedSearchParam'
+
+export const RoflAppInstanceDetailsPage: FC = () => {
+  const { t } = useTranslation()
+  const scope = useRequiredScopeParam()
+  const id = useParams().id!
+  const rak = useParams().rak!
+  const txLink = useHref('')
+  const [method, setMethod] = useTypedSearchParam('method', 'any', {
+    deleteParams: ['page'],
+  })
+  const context: RoflAppInstanceDetailsContext = { scope, id, rak, method, setMethod }
+  const instancesQuery = useGetRuntimeRoflAppsIdInstances(scope.network, Layer.sapphire, id, {
```

Comment:
This seems highly inefficient. And I could see potential problem with pagination i.e. instances not being in the page you are querying.

Reply:
yes, I described this in PR description. For now there is no chance we will hit an issue, because number of instances is very low per app.

Reply:
New endpoint was added in Nexus, need to bump PR